Black-Market/Black-Market/Black_MarketApp.swift

```diff
@@ -11,7 +11,7 @@ import SwiftUI
 struct Black_MarketApp: App {
     var body: some Scene {
         WindowGroup {
-            ContentView()
+            DemoComponentsView()
```

Comment:
i wouldn’t commit this view into main as is just a demo for the components

Reply:
Yes, in the next PR I will change it to SignIn flow.